tests/kernel/poll/src/test_poll.c

```diff
@@ -344,8 +382,10 @@ void test_poll_wait(void)
 	check_results(&wait_events[1], K_POLL_TYPE_DATA_AVAILABLE, false);
 	check_results(&wait_events[2], K_POLL_TYPE_SIGNAL, false);
 
-	/* fifo */
-	rc = k_poll(wait_events, ARRAY_SIZE(wait_events), K_SECONDS(1));
+	/* fifo, Time out can set to K_FOREVER since
+	 * main thread is high priority.
+	 */
+	rc = k_poll(wait_events, ARRAY_SIZE(wait_events), K_FOREVER);
```

Comment:
My understanding is that you think that "K_SECONDS(1)" should be replaced with "K_FOREVER".

Reply:
I don't see a good reason to change this code.  What's the benefit of eliminating the timeout?

Reply:
As I re-check the test code,  It's better to align with context,  so  I revert my change in this place.